content/en/docs/concepts/extend-kubernetes/api-extension/_index.md

```diff
@@ -1,4 +1,14 @@
 ---
 title: Extending the Kubernetes API
 weight: 20
+content_type: concept
 ---
+
+
+<!-- Overview -->
+
+A _Custom Resource Definition_ (CRD) defines the name and structure of a custom object type that you want to allow in your Kubernetes cluster. A CRD is a way of defining what the objects will be called along with with properties it can possess like _size, color-_ etc.
```

Comment:
```suggestion
A _Custom Resource Definition_ (CRD) defines the name and structure of a custom object type that you want to allow in your Kubernetes cluster. A CRD is a way of defining what the objects will be called along with properties it can possess like _size, color-_ etc.
```